src/app/error.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+import React, { useState } from "react";
+import Button from "@mui/material/Button";
+import {
+    ErrorCenterer,
+    ErrorPanel,
+    ErrorLargeText,
+    ErrorSecondaryText,
+    ErrorRetryText,
+} from "./errorPageStyling";
+
+interface ErrorProps {
+    reset: () => void;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const Error: React.FC<ErrorProps> = ({ reset }) => {
+    const [ErrorMessage, SetErrorMessage] = useState("");
+
+    const retry = (): void => {
+        SetErrorMessage("Retrying...");
+        setTimeout(() => {
```

Comment:
Why wait for this?

Reply:
So it has the text onscreen for a second, however this is after the error page rerenders, so should probably change the text to make it more clear.